docs/app-template.md

```diff
@@ -0,0 +1,10 @@
+# 
```

Comment:
Let's have this say `# AppName`

Reply:
I removed it for quick copy and paste. Its fast for me to just add the actual app name.

Reply:
Only reason I think it should keep the appname is as an example to others in the future.